landingzones/tests/test_views.py

```diff
@@ -163,6 +165,110 @@ def test_render(self):
         self.assertIsNotNone(form.fields['configuration'])
 
 
+class TestLandingZoneUpdateView(TestViewsBase):
+    """Tests for the landing zone update view"""
+
+    def test_render(self):
+        """Test rendering of the landing zone update view"""
+        with self.login(self.user):
+            response = self.client.get(
+                reverse(
+                    'landingzones:update',
+                    kwargs={'landingzone': self.landing_zone.sodar_uuid},
+                )
+            )
+        self.assertEqual(response.status_code, 200)
+        # Assert form
+        form = response.context['form']
+        self.assertIsNotNone(form)
+        self.assertIsNotNone(form.fields['assay'])
+        self.assertIsNotNone(form.fields['description'])
+        # Make sure to also assert the expected fields are hidden with the HiddenInput widget.
+        self.assertIsInstance(form.fields['title_suffix'].widget, HiddenInput)
+        self.assertIsInstance(form.fields['configuration'].widget, HiddenInput)
+        self.assertIsInstance(form.fields['create_colls'].widget, HiddenInput)
+        self.assertIsInstance(form.fields['restrict_colls'].widget, HiddenInput)
+        self.assertIsInstance(form.fields['assay'].widget, HiddenInput)
+
+    def test_render_invalid_status(self):
+        """Test rendering with an invalid zone status"""
+        self.landing_zone.status = 'DELETED'
+        self.landing_zone.save()
+
+        with self.login(self.user):
+            response = self.client.get(
+                reverse(
+                    'landingzones:update',
+                    kwargs={'landingzone': self.landing_zone.sodar_uuid},
+                )
+            )
+        self.assertEqual(response.status_code, 302)
+
+    def test_post(self):
+        """Test POST request to the landing zone update view"""
+        with self.login(self.user):
+            response = self.client.post(
+                reverse(
+                    'landingzones:update',
+                    kwargs={'landingzone': self.landing_zone.sodar_uuid},
+                ),
+                data={
+                    'assay': self.assay.sodar_uuid,
+                    'description': 'test description updated',
+                    'user_message': 'test user message',
+                },
+            )
+            self.assertRedirects(
+                response,
+                reverse(
+                    'landingzones:list',
+                    kwargs={'project': self.project.sodar_uuid},
+                ),
+            )
+        landing_zone = LandingZone.objects.get(
+            sodar_uuid=self.landing_zone.sodar_uuid
+        )
+        self.assertEqual(
+            landing_zone.assay,
+            self.assay,
```

Comment:
I think there are some unnecessary line breaks in these asserts. hint: if you set the arguments on the same line with no comma after the second one, black won't force them on multiple lines.

It's again a very minor point, but these test files can get huge so let's keep them as concise as we can :)